myimdb/wsgi.py

```diff
@@ -0,0 +1,20 @@
+"""
+WSGI config for myimdb project.
+
+It exposes the WSGI callable as a module-level variable named ``application``.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.9/howto/deployment/wsgi/
+"""
+
+import os
+
+from django.core.wsgi import get_wsgi_application
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "myimdb.settings")
+
+application = get_wsgi_application()
+
+from whitenoise.django import DjangoWhiteNoise
```

Comment:
whitenoise is used to populate the requirements.txt file with relevant data.
https://djangogirls.gitbooks.io/django-girls-tutorial-extensions/content/heroku/